### PR TITLE
fix: align elements, replace img for next/image and anchors for next/…

### PR DESCRIPTION
**Pull request by mateosierra17**

### CODE REVIEW

File: components/Header/index.tsx [hunk 1]
```diff
@@ -2,20 +2,31 @@ import React from 'react'
...
-          About
-        </Link>
+        {mapLinks}
       </nav>
-      <h1 className="text-2xl font-bold hover:text-yellow-600">
+      <h1 className="home text-2xl font-bold hover:text-yellow-600 text-center">
         <Link href="/">Synergy Wire Blog </Link>
       </h1>
-      <div>
+      <div className='header-socials'>
```
Comment (calixfar) [inline components/Header/index.tsx]: Use double quote instead of single one

File: components/Header/index.tsx [hunk 2]
```diff
@@ -2,20 +2,31 @@ import React from 'react'
...
 import Icon from '../icons/Icon'
 import { ICON_KEYS } from '@/utils/contants'
+import { HeaderLink } from '@/interfaces/UI'
 import './header.css'
 
 export default function Header() {
+  const HeaderLinks: HeaderLink[] = [
+    {
+      text: 'About',
+      href: './about'
```
Comment (calixfar) [inline components/Header/index.tsx]: Delete the .

File: components/Header/index.tsx [hunk 3]
```diff
@@ -26,19 +37,12 @@ export default function Header() {
       <label className="responsive-menu" htmlFor="toggle-button">
         <input type="checkbox" id="toggle-button" />
         <div className="responsive-menu__icon-switch">
-        <Icon  name={ICON_KEYS.MENU} ></Icon>
+          <Icon name={ICON_KEYS.MENU} ></Icon>
           <Icon name={ICON_KEYS.CLOSE} ></Icon>
```
Comment (calixfar) [inline components/Header/index.tsx]: Use single row syntax:

<Icon name={...} />

File: components/PostPreview/PostPreview.tsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { PostMetaData } from '@/interfaces/PostMetaData'
 import './styles.css'
 
 export function PostPreview(props: PostMetaData) {
+  console.log(props.tags)
```
Comment (calixfar) [inline components/PostPreview/PostPreview.tsx]: Remove console.log. I know, this is mine but you are adding it on this PR.

File: next.config.js [hunk 5]
```diff
@@ -3,6 +3,16 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
+  images: {
```
Comment (calixfar) [inline next.config.js:6]: What is this for?

**Review by calixfar — CHANGES_REQUESTED** [review]

File: components/Header/index.tsx (hunk 1 above)
Comment (mateosierra17) [reply components/Header/index.tsx]:
```suggestion
      <div className="header-socials">
```

**Review by calixfar — APPROVED** [review]

**Pull request merged**